src/homepage/Running_SoS.ipynb

```diff
@@ -216,12 +216,7 @@
    "source": [
     "Install Bash Jupyter kernel using command\n",
     "\n",
-    "```\n",
-    "pip install bash_kernel\n",
-    "python -m bash_kernel.install\n",
-    "```\n",
-    "\n",
-    "or following instructions from [Jupyter Bash Kernel homepage](https://github.com/takluyver/bash_kernel). You then need to install the `sos-bash` module to connect the kernel to SoS\n",
+    "Please follow the instructions from [Jupyter Bash Kernel homepage](https://github.com/takluyver/bash_kernel). You then need to install the `sos-bash` module to connect the kernel to SoS\n",
```

Comment:
"please follow .... to install ....", right? You cannot just say "follow instruction" without an aim. Sometimes you also need to say, please install (e.g. julia)](link to julia)) and then the jupyter kernel for julia following instructions from [here](....).

Reply:
Yes, your sentences did not read complete to me. You do not even have to say following instruction, just something like “Install [bash kernel](url) for Jupyter, checking if bash appears in the output of `jupyter kernelspec`, then..”. You can use different sentences for different languages to avoid repetition.

Reply:
Understood!

Reply:
In the original documentation, there are some sentences to ask the users to check whether the kernel is installed properly by `jupyter kernelspec list`. However, it only appears in some languages. Do you want to include it in all the languages? Or just some of them.

Reply:
I would add a general description under "Supported Languages" something like this (revise it please):

To use a language in SoS, your system should have the corresponding interpreters (bash, python, shell) and jupyter kernels (e.g. ir for R, ijavascript for JavaScript) installed. The installation process varies from language to language and can be quite troublesome depending on language and operating systems. Generally speaking, before installing a SoS language module, you should
1. Verify the accessibility of an interpreter by executing the interpreter from a command line (e.g. run `julia` to check if `julia` is installed.
2. Verify if the corresponding kernel is installed correctly by running 
```
jupyter kernelspec list
```
3. Start jupyter, create a notebook with the kernel and check if you can use the kernel correctly

You can then following the instructions in the following subsections to installed required libraries or modules, and then the sos language modules. The last step is usually as trivial as `pip install sos-r`.

Reply:
You can then remove the individual steps from the subsections.

Reply:
Cool! So in each subsections (of language), I need to say (for example):

```
To use this kernel, please follow {language name} kernel homepage to install {language name} kernel.
```
After you verify that {language name} is working, you will need to install the sos-javascript module with command

pip install sos-{language name}

Reply:
Yeap, something like that.

Reply:
But for some languages such as `MATLAB` and `R`, I think what you wrote is still useful. Do you want to keep them or delete them?

Reply:
Please refrain from deleting all details. We should keep the troubleshooting part and module installation parts, and only remove duplicated instructions such as `jupyter kernelspec list'.